README.md

```diff
@@ -0,0 +1,22 @@
+# Aries Mobile Agent Android
+
+## Installation
+* Add Hyperledger Maven repo `maven { url "https://repo.sovrin.org/repository/maven-public" }` in build.gradle
+* Set minSdkVersion to 21 in build.gradle
```

Comment:
Probably also need to set the ndk version, since we're tied to the libc++_shared.so version you pulled from your ndk install?